### DISCUSSION TITLE
Please help a newbie

### DISCUSSION THREAD

**Discussion by sprs123** [AlexandreRouma/SDRPlusPlus#823] [General]
I am trying to load SDR++ onto Ubuntu MATE 22.04./ I get the .deb file (1.0.4/sdrpp_ubuntu_focal_amd64.deb)  downloaded and opened and installed (get an icon under menu) but when I open the icon absolutely nothing happens. I would appreciate any help but PLEASE baby steps and I mean baby steps (old and stupid), many thanks

**Comment by AlexandreRouma** [comment]
wrong deb file. Use the `jammy` deb. As that version of ubuntu came out after I released 1.0.4, you will have to download the much more recent nightly version https://www.sdrpp.org/nightly.

**Reply by sprs123** [reply]
Hi Alexandre

Sorry to hassle you again but if you have a minute free... please help. I downloaded and installed the jammy nightly version and it compiles but when i click on the icon it opens for half a second then nothing, what have I done wrong, cheers

**Reply by AlexandreRouma** [reply]
run it from a terminal and show what it says. You probably haven't installed the dependencies listed in the readme.

**Reply by sprs123** [reply]
Really sorry to hassle you like this but I am old and stupid and you are young and bright, so any help appreciated. When I download the file it works on my Windows computer but not on my Ubuntu MATE computer (the link is not live). So i copied it across and extracted it. Then there are two files 1) Debian and 2) usr how do I install from this. really sorry to take up your time, cheers

**Reply by sprs123** [reply]
I have already installed all the dependencies listed in the readme, and just done it again but still no luck

**Reply by byggemandboesen** [reply]
To gather some more insight into the problem it would be very helpful with some images and etc. showing the problem you're encountering. As already mentioned, try running SDR++ directly from the terminal and share the output😉

**Reply by sprs123** [reply]
Hi byggemandboesen
thanx for feedback here is the output

![Screenshot at 2022-07-28 08-50-39](https://user-images.githubusercontent.com/110127530/181440070-a334700e-7d92-4bed-a5d0-ac96f287c4d1.png)
![Screenshot at 2022-07-28 08-50-39](https://user-images.githubusercontent.com/110127530/181440497-56115fc4-171b-441f-9202-f827c432c47b.png)

richad@richad-K52F:~$ sdrpp
[2022-07-28 08:48:53.959] [info] SDR++ v1.1.0
[2022-07-28 08:48:53.959] [info] Loading config
[2022-07-28 08:48:53.960] [warning] ConfigManager locked, waiting...
[2022-07-28 08:48:54.070] [error] Glfw Error 65543: GLX: Failed to create context: GLXBadFBConfig
[2022-07-28 08:48:54.070] [info] OpenGL 3.0 was not supported
[2022-07-28 08:48:54.078] [error] Glfw Error 65543: GLX: Failed to create context: GLXBadFBConfig
[2022-07-28 08:48:54.078] [info] OpenGL 3.1 ES was not supported
[2022-07-28 08:48:54.149] [info] Using OpenGL 2.1
Failed to initialize OpenGL loader!
[2022-07-28 08:48:54.335] [warning] Could not init using OpenGL with normal GLSL version, falling back to GLSL 1.2
Failed to initialize OpenGL loader!
[2022-07-28 08:48:54.335] [error] Failed to initialize OpenGL with GLSL 1.2
richad@richad-K52F:~$ 


sorry not sure how to attach the actual picture, any help appreciated, cheers

**Reply by byggemandboesen** [reply]
Very helpful images! It looks like it's due to issues with the graphics driver on your system. How old is your system? Have you installed the necessary graphics drivers on your machine?

**Reply by sprs123** [reply]
Hi Victor
It is an old computer but new install of Ubuntu MATE and apart from that I have not really up-loaded any drivers. I am actually going to give up on that computer as it keeps dying, so thanks for your input but I will carry on using my RPi4. Does sdr++ compile on the RPi4 (bullseye) and if so which version, (the nightly version???) thx richrd

**Reply by byggemandboesen** [reply]
In that case the pc is probably too old or without the proper drivers. To install SDR++ on the raspberry pi, simply get the `sdrpp_raspios_bullseye_armhf` deb file from the nightly builds and then run `sudo dpgk -i sdrpp_raspios_bullseye_armhf.deb` from the directory with the deb file like you have probably also done on your linux machine.
I hope you get it working!

**Comment by fizr5** [comment]
The computer is not too old. The computer is perfectly supporting OpenGL 2.1. Its first gen Intel Core i series with "Intel HD" graphics. The issue here is the developer of the SDR++ software not even properly testing the software on his own computer even when he could have done this for years. More information here:
https://github.com/AlexandreRouma/SDRPlusPlus/issues/1325

**Comment by W5JAN** [comment]
I am also having issues loading SDRPP (aka SDR++) on my Ubuntu Linux 22.04 system.  The three tripping points seem to be a failure to find libvolk2-dev, librtaudio-dev, and libzsttd-dev.  In a way it has been a good thing because I am being reconnected with the use of terminal,  but I am still striking out.  Can someone help me?  I suspect the fix is something I have overlooked or failed to do.

Regards
John Noffke  

jnoffke@jnoffke-Aspire-TC-780:~/Downloads$ sudo dpkg -i sdrpp_ubuntu_jammy_amd64.deb
[sudo] password for jnoffke: 
(Reading database ... 216298 files and directories currently installed.)
Preparing to unpack sdrpp_ubuntu_jammy_amd64.deb ...
Unpacking sdrpp (1.2.0-1386) over (1.2.0-1386) ...
dpkg: dependency problems prevent configuration of sdrpp:
 sdrpp depends on libvolk2-dev; however:
  Package libvolk2-dev is not installed.
 sdrpp depends on librtaudio-dev; however:
  Package librtaudio-dev is not installed.
 sdrpp depends on libzstd-dev; however:
  Package libzstd-dev is not installed.

**Comment by njjeff201** [comment]
Hi New guy in town just received my SDR V4 for an iMAC.  Could someone please steer me in the right direction to get started.  Is there a 'list' somewhere so I am aware of what is needed?  Thank you, Jeff

**Reply by fizr5** [reply]
Install Linux on the computer(iMAC) and install SDR++ from the Linux repository.

**Reply by njjeff201** [reply]
I finally got SDR++ installed on the iMAC but there is no audio out.  Do you still think running Linux is the way to go?  SDR Angel is also installed.
Thanx!

**Reply by fizr5** [reply]
Yes, try out for example DragonOS. It have already everything SDR related build in: https://sourceforge.net/projects/dragonos-focal/files/
